mesos.sh

```diff
@@ -2,12 +2,16 @@ package: mesos
 version: v1.11.0
 tag: 1.11.0
 source: https://git-wip-us.apache.org/repos/asf/mesos.git
+requires:
+- zlib
+- curl
```

Comment:
@teo Mesos is only package that requires our own `curl` on CS8, could this be replaced with:
```
  - "system-curl:(slc8)" 
  - "curl:(?!slc8)"
```
?